### PR TITLE
Fast-follows for merge jobs

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

- Add callout to CI/CD guide https://docs.getdbt.com/guides/custom-cicd-pipelines?step=1 that points to Merge jobs page
- Update prerequisites for Merge jobs

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] Needs review from PM

### CODE REVIEW

File: website/docs/docs/deploy/merge-jobs.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ By using CD in dbt Cloud, you can take advantage of deferral to build only the e
 ## Prerequisites
 - You have a dbt Cloud account. 
 - You have access to the beta release for this functionality. 
+- You have set up a [connection with your Git provider](/docs/cloud/git/git-configuration-in-dbt-cloud). This integration lets dbt Cloud run jobs on your behalf for job triggering.
```
Comment (nghi-ly) [inline website/docs/docs/deploy/merge-jobs.md:14]: double-checking: any other prereqs for Merge jobs that we should add?

File: website/docs/guides/custom-cicd-pipelines.md [hunk 2]
```diff
@@ -509,7 +515,7 @@ Additionally, you’ll see the job in the run history of dbt Cloud. It should be
 
 If your git provider is not one with a native integration with dbt Cloud, but you still want to take advantage of CI builds, you've come to the right spot! With just a bit of work it's possible to setup a job that will run a dbt Cloud job when a pull request (PR) is created.
 
-:::info Run on PR
+:::tip Run on PR
 
 If your git provider has a native integration with dbt Cloud, you can take advantage of the setup instructions [here](/docs/deploy/ci-jobs).
```
Comment (nghi-ly) [inline website/docs/guides/custom-cicd-pipelines.md:520]: for consistency and parallel construction, new callout on line 61 follows this ^ format

File: website/docs/docs/deploy/merge-jobs.md [hunk 3]
```diff
@@ -4,15 +4,16 @@ sidebar_label: "Merge jobs"
 description: "Learn how to trigger a dbt job run when a Git pull request merges."
 ---
 
-# Merge jobs in dbt Cloud <Lifecycle status="beta" />
+# Merge jobs in dbt Cloud
```
Comment (nghi-ly) [inline website/docs/docs/deploy/merge-jobs.md]:
```suggestion
```

File: website/docs/docs/deploy/merge-jobs.md (hunk 1 above)
Comment (reubenmc) [reply website/docs/docs/deploy/merge-jobs.md:14]: Maybe the stuff around ensuring your git provider is set to send push notifications and link to the other documentation? Although that might be overkill for here.

**Review by joellabes — APPROVED** [review]

File: website/docs/docs/deploy/merge-jobs.md [hunk 4]
```diff
@@ -4,15 +4,16 @@ sidebar_label: "Merge jobs"
...
 
 By using CD in dbt Cloud, you can take advantage of deferral to build only the edited model and any downstream changes. With merge jobs, state will be updated almost instantly, always giving the most up-to-date state information in [dbt Explorer](/docs/collaborate/explore-projects).
 
 ## Prerequisites
 - You have a dbt Cloud account. 
-- You have access to the beta release for this functionality. 
+- You have set up a [connection with your Git provider](/docs/cloud/git/git-configuration-in-dbt-cloud). This integration lets dbt Cloud run jobs on your behalf for job triggering.
+   - If you're using a native [GitLab](/docs/cloud/git/connect-gitlab) integration, you need a paid or self-hosted account that includes support for GitLab webhooks and [project access tokens](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html). If you're using GitLab Free, merge requests will trigger CI jobs but CI job status updates (success or failure of the job) will not be reported back to GitLab.
```
Comment (mirnawong1) [inline website/docs/docs/deploy/merge-jobs.md:15]: active voice sugg
```suggestion
   - If you're using a native [GitLab](/docs/cloud/git/connect-gitlab) integration, you need a paid or self-hosted account that includes support for GitLab webhooks and [project access tokens](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html). If you're using GitLab Free, merge requests will trigger CI jobs but GitLab won't report back the CI job status updates (success or failure of the job).
```

File: website/docs/guides/custom-cicd-pipelines.md [hunk 5]
```diff
@@ -58,6 +58,12 @@ Here’s a quick look at what this pipeline will accomplish:
 
 This job will take a bit more to setup, but is a good example of how to call the dbt Cloud API from a CI/CD pipeline. The concepts presented here can be generalized and used in whatever way best suits your use case.
 
+:::tip Run on merge
+
+If your Git provider has a native integration with dbt Cloud, you can take advantage of setting up [Merge jobs](/docs/deploy/merge-jobs) in the UI.
```
Comment (mirnawong1) [inline website/docs/guides/custom-cicd-pipelines.md:63]: 🤩

**Review by mirnawong1 — APPROVED** [review]
looks awesome, thanks @nghi-ly and small non blocker from me!

File: website/docs/docs/deploy/merge-jobs.md (hunk 4 above)
Comment (nghi-ly) [reply website/docs/docs/deploy/merge-jobs.md:15]: like your sugg but it changes the meaning of the sentence (updates not reported to GitLab vs GitLab not reporting updates)

**Pull request merged**